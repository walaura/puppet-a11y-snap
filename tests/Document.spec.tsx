import React from "react";
import Document from "./Document";
import getSnap, { utils, close } from "../src/index";

const { getHeadings } = utils;

test("Form flat snapshot to make sense", async () => {
  const component = <Document />;
  const snap = await getSnap(component);
  console.log(getHeadings(snap));
  expect(snap).toMatchSnapshot();
});

afterAll(() => {
  close();
});
