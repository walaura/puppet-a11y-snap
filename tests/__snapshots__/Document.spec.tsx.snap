// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form flat snapshot to make sense 1`] = `
Array [
  Object {
    "children": Array [
      Object {
        "children": Array [
          Object {
            "name": "Lorem ipsum",
            "role": "text",
          },
        ],
        "level": 1,
        "name": "Lorem ipsum",
        "role": "heading",
      },
      Object {
        "children": Array [
          Object {
            "name": "What is Lorem Ipsum?",
            "role": "text",
          },
        ],
        "level": 2,
        "name": "What is Lorem Ipsum?",
        "role": "heading",
      },
      Object {
        "children": Array [
          Object {
            "name": "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
            "role": "text",
          },
        ],
        "name": "",
        "role": "paragraph",
      },
      Object {
        "children": Array [
          Object {
            "name": "It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.",
            "role": "text",
          },
        ],
        "name": "",
        "role": "paragraph",
      },
      Object {
        "children": Array [
          Object {
            "name": "Why do we use it?",
            "role": "text",
          },
        ],
        "level": 2,
        "name": "Why do we use it?",
        "role": "heading",
      },
      Object {
        "children": Array [
          Object {
            "name": "It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum'",
            "role": "text",
          },
        ],
        "name": "",
        "role": "paragraph",
      },
    ],
    "name": "",
    "role": "article",
  },
]
`;
